### PR TITLE
feat(metrics): Improve metrics inventory performance and support large scale clusters

### PR DESCRIPTION
**Pull request by Noy-Simon**
public: Improve metrics inventory performance and support large scale clusters

### CODE REVIEW

File: charts/komodor-agent/values.yaml
Comment (nirbenator) [inline charts/komodor-agent/values.yaml:1]: @Noy-Simon - if we're removing metrics from the komodor-agent (or the deployment), id move metrics values  from components.komodorAgent.metrics to componets.komodorDaeon.metrics

Comment (Noy-Simon) [reply charts/komodor-agent/values.yaml:1]: @nirbenator  done

File: charts/komodor-agent/templates/metrics/configmap-daemon.yaml [hunk 1]
```diff
@@ -33,5 +33,20 @@ data:
       namedrop = ["kubernetes_system_container"]
       [inputs.kubernetes.tags]
         measure_type = "usage"
+        
+    [[inputs.kube_inventory]]
+      ## URL for the Kubernetes API
+      url = {{ .Values.communications.apiServerUrl | quote }}
+      url_kubelet = "https://$NODE_IP:10250"
```
Comment (Moran-k) [inline charts/komodor-agent/templates/metrics/configmap-daemon.yaml:40]: Is this port always 10250? Don't we need it to be configurable?

**Review by Moran-k — APPROVED** [review]
Looks good to me

File: charts/komodor-agent/templates/metrics/configmap-daemon.yaml (hunk 1 above)
Comment (galHalup) [reply charts/komodor-agent/templates/metrics/configmap-daemon.yaml:40]: it's the kubelet port, I guess it won't be changed soon

**Pull request merged**